src/path_converters.h

```diff
@@ -1015,9 +1018,10 @@ class Sketch
           m_last_y(0.0),
           m_has_last(false),
           m_p(0.0),
-          m_rand(0)
+          m_rand(0),
+          m_seed0(seed)
     {
-        rewind(0);
+        rewind(0); // re-seeds PRNG with m_seed0 again
```

Comment:
Well it is there since:
https://github.com/matplotlib/matplotlib/commit/e1e0848625a7802f4398a0c599fbd0ffd06bded3

I had decided not to meddle with it since I did not fully understand it's purpose .